packages/data/src/useWalkthroughData.ts

```diff
@@ -75,7 +75,7 @@ export interface QuestionMultipleChoiceData extends QuestionBaseData {
 
 export const WalkthroughItemTypeMultiChoiceMultiple = "multiChoiceMultiple";
 export const isWalkthroughItemTypeMultiChoiceMultiple = (
-  walkthroughItemType: string,
+  walkthroughItemType: string
```

Comment:
My prettier puts this back in...

Can you run `npm run format` from the root? That _**should**_ format everything appropriately.